app/src/features/surveys/components/GeneralInformationForm.tsx

```diff
@@ -169,7 +169,7 @@ const GeneralInformationForm: React.FC<IGeneralInformationFormProps> = (props) =
           />
         </Grid>
         <Grid item xs={12}>
-          <FormControl fullWidth variant="outlined" style={{ width: '100%' }}>
+          <FormControl fullWidth variant="outlined" required={true} style={{ width: '100%' }}>
```

Comment:
question: we decided that methodology is gonna be a required field now?

Reply:
is this change only gonna be on fe side and the model will still allow it to be optional (not null) for now?

Reply:
We just added it to the frontend for now, because without picking a methodology, none of the validation runs correctly (as that is a required piece in determining the template/validation schema)

Reply:
But, the whole template/methodology stuff is likely going to evolve a bit, so simple fix for now is best